Assignment/Program.cs

```diff
@@ -1,7 +1,21 @@
 ﻿namespace Assignment {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+         Console.Write ("Enter total votes : ");
+         string input = Console.ReadLine ()!.ToLower ().Replace (" ", "");

```

Comment:
Fix this exception.
![image](https://github.com/YoganathanM/Assignment/assets/94427955/6874a840-157c-4729-84d6-f37a0d5b6366)

Reply:
Done